resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -232,6 +149,13 @@ private[spark] class KubernetesSuite extends SparkFunSuite
     }
 
     Eventually.eventually(TIMEOUT, INTERVAL) {
+      val driverPod = kubernetesTestComponents.kubernetesClient
```

Comment:
Why you have to add this?

Reply:
Will double check If I saw an issue or was a left over from a refactoring.

Reply:
Will remove it.